### PR TITLE
Update library's build.gradle to support Artifactory publishing

### PR DESCRIPTION
**Pull request by elsnosrap**
This is required to set up an automated build / deploy job for this library.

### CODE REVIEW

File: library/build.gradle [hunk 1]
```diff
@@ -23,3 +35,35 @@ dependencies {
...
+        }
+    }
+}
+
+publishing {
+    publications {
+        ViewPagerIndicator(MavenPublication) {
+            groupId 'com.viewpagerindicator'
+            artifactId 'ViewPagerIndicator'
+            version '2.5.1'
```
Comment (elsnosrap) [inline library/build.gradle]: I initially had this set to:
```
version "$android.versionName"
```
but it didn't work. I received an error when uploading to Artifactory that the version already existed (which it didn't). I have to believe there's a way to get this to work, but w/e.

File: library/build.gradle [hunk 2]
```diff
@@ -8,6 +19,7 @@ android {
         minSdkVersion 17
         targetSdkVersion 23
         versionCode 1
+        // NOTE: When updating versionName, be sure to update version in publishing group below too
```
Comment (firekesti) [inline library/build.gradle]: could we make this a variable so we only need to update it once? I know gradle's scopes are tricky; if you already tried that and it was weird/tough, we can easily follow a comment's directions!

Comment (elsnosrap) [reply library/build.gradle]: ugh, so I added the comment below right after I made this PR. However, I forgot to hit "standalone comment" and instead started a review. Please refer to the now-published comment.

Comment (elsnosrap) [reply library/build.gradle]: I'm going to try using a variable one more time before I give up and just merge this in.

**Review by firekesti — APPROVED** [review]

File: library/build.gradle (hunk 2 above)
Comment (elsnosrap) [reply library/build.gradle]: @firekesti I fixed this. The version is only declared once now.

**Pull request merged**